src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -444,6 +448,26 @@ protected function getEntitiesByModule()
         }
         return $entitiesByModule;
     }
+    
+    /**
+     * Loads all modules and themes installed, including app. Uses the format of
+     * the @link ModuleLoader manifest for themes as well.
+     * Themes can be references with "themes/{theme name}".
+     */
+    private function getModulesAndThemes(): array
```

Comment:
This new method is called _four times_ in this collector - it should probably store the result in a `private` property the first time, and then just return that value the other three times.

Alternatively this should set a property, and only ever be called once. Then just directly reference the property all the other times. If you use this approach, the method should probably be called something like `findModulesAndThemes()` instead.

Reply:
went with option 1. done.